Makefile

```diff
@@ -1,9 +1,9 @@
 
 usage:
-	@echo "make all    - go-vet, test and build project"
-	@echo "make build  - build project"
-	@echo "make test   - run unit tests"
-	@echo "make go-vet - go vet"
+	@echo "make all                  - go-vet, test and build project"
+	@echo "make version=v1.0.0 build - build project"
```

Comment:
I'd set the version to 0.1.0 as we literally have nothing content wise at the moment.